File: scripts/parse_pdf.py
import fitz

def parse_pdf():
    try:
        doc = fitz.open('Accounting.pdf')
        text = ""
        for page in doc:
            text += page.get_text() + "\n"
        with open('Accounting.txt', 'w', encoding='utf-8') as out:
            out.write(text)
        print("Success")
    except Exception as e:
        print(f"Error: {e}")

if __name__ == "__main__":
    parse_pdf()

File: scripts/parse-pdf.js
import fs from 'fs';
import pdf from 'pdf-parse';

async function parse() {
    try {
        const dataBuffer = fs.readFileSync('Accounting.pdf');
        const data = await pdf(dataBuffer);
        fs.writeFileSync('Accounting.txt', data.text);
        console.log("PDF text extracted to Accounting.txt");
    } catch (err) {
        console.error("Error:", err);
    }
}

parse();
